server/auth/emailTransporter.js

```diff
@@ -0,0 +1,36 @@
+const nodemailer = require('nodemailer');
+const nodemailermock = require('nodemailer-mock');
+
+// create reusable transporter class using the default SMTP transport with functions
+function createTransport() {
+  // nodeMailer options
+  const options = {
+    host: process.env.SMTP_HOST,
+    port: process.env.SMTP_PORT,
+    auth: {
+      user: 'user ',
+      pass: 'pass',
+    },
+  };
+  // mock Transporter in testing env
+  const transporter = process.env.NODE_ENV === 'test' ? nodemailermock.createTransport(options) : nodemailer.createTransport(options);
+  return transporter;
+}
+// send mail with defined transport object
+function sendMail(transporter, recipient, subject, content) {
+  const mailOptions = {
+    from: process.env.NODEMAIL_EMAIL,
+    to: recipient,
+    subject: subject,
+    text: content,
+  };
+  transporter.sendMail(mailOptions, (error, info) => {
+    if (error) {
+      console.log(error);
+    } else {
+      console.log('Email sent: ' + info.response);
+    }
+  });
+}
+
```

Comment:
These functions are very generic to email sending. The auth folder doesn't quite feel like the right place for it, what do you think?

Reply:
Definitely agree with you. I left it in there as I was not certain of the best place to put it. Would love your input/thoughts on best placement!

Reply:
I'm actually wondering if we should have a new directory like `/server/mailer` to contain this file, and any additional email related files that we may end up adding. 

For example, I'd like to consider using https://www.npmjs.com/package/email-templates to create nicer looking HTML emails. So we might end up having a `/server/mailer/views` directory with text and html ejs templates that we can use, etc.